### ISSUE TITLE
Carousel - accessibility issues for screenreader useres

### ISSUE THREAD

**Issue by JuliaSang** [w3c/w3c-website#2]
Raised by the Digital Accessibility Centre as part of their accessibility testing: 

The carousel presented accessibility issues to our screen reader analyst.		

WCAG Reference:
4.1.2 Name, Role, Value (Level A)
Understanding Name, Role, Value | How to Meet Name, Role, Value

URL: https://w3c-dev.studio24.dev/business/index.html

The carousel had been updated and new issues were presented to screen reader users, the previous and next buttons do not function with VoiceOver running on an iOS mobile device.

Current code ref(s):
```
<figure class="slide js-current" role="group" aria-roledescription="slide" aria-label="Slide 1 of 3" style="left: 0px;">
<blockquote class="slide__quote">
<p>The School of Informatics at the University of Edinburgh is a world leader in research and teaching in Computer Science, Artificial Intelligence and Cognitive Science.</p>
<p>W3C technologies are both used and studied here, and we are proud to contribute to their development.</p>
</blockquote>
<figcaption class="slide__cite l-sidebar">
<div>
<div class="sidebar">
<div class="slide__logo">
<img src="dist/assets/images/edinburgh-uni.png" alt="The University of Edinburgh" loading="lazy">
</div>
</div>
<div class="not-sidebar">
<p>Dierdre De Veloper</p>
<p>Lead Dev</p>
</div>
</div>
</figcaption>
</figure>
```

Screen reader comments:
“The group does not announce with NVDA but I am advised when I enter the slide. The slide number does however announce when I use the next or previous buttons.
The group or slide does not announce with VoiceOver.”

New issues:
“With VoiceOver, I select the ‘Next slide’ button however the slide does not change. I would not be able to move to the next slide with this software.

With JAWS with Microsoft Edge Chromium when I selected the next or previous slide buttons my focus did not move to the slide but elsewhere on the page which was disorientating. This did not occur with other software or with JAWS and Google Chrome.”

General conclusion:
“The carousel is more problematic to use than the previous time I tested the functionality. It is now inaccessible with VoiceOver as the next and previous buttons do not perform an action.

Focus is more problematic with JAWS and Edge Chromium which would be disorientating for screen reader users.”

**Solution:**
It is advised the previous version of the carousel is used but remove the aria-label from the unordered list as previously suggested.
The use of the role of group and the announcement of the aria-roledescription was of benefit however the role of group is not supported by VoiceOver on iOS.

**Comment by NicolaSaunders** [comment]
Addressed by https://github.com/w3c/w3c-website-redesign-html/commit/06c4bf9848197b990668940d865feb7c78da4a04
